tinkerbell/stack/README.md

```diff
@@ -103,3 +117,9 @@ hegel:
   hegel:
     image: quay.io/tinkerbell/hegel:latest
 ```
+
+### Boots Parameters
+
+| Name | Description | Value |
+| ---- | ----------- | ----- |
+| `boots.boots.hostNetwork` | Whether to deploy Boots using `hostNetwork` on the pod spec. When `true` Boots will be able to receive DHCP broadcast messages. If `false`, Boots will be behind the load balancer VIP and will need to receive DHCP requests via unicast. | `true` |  
```

Comment:
What's with the double `boots`?

Reply:
the first `boots` is the dependency name from `Chart.yaml` file. The second `boots` is the key that is imported in the sub-chart. Helm doesn't allow importing all top level variables of a sub-chart so I put all variables under `boots` in the sub-chart to make it easier to import.